src/NumFields.jl

```diff
@@ -21,6 +21,8 @@ canonical_unit(a::NumFieldElem) = a
 
 characteristic(F::NumField) = 0
 
+is_known(::typeof(characteristic), F::NumField) = true
```

Comment:
This could be removed again if we go with the fallback approach I suggest. Otherwise we should keep it.